### DISCUSSION TITLE
Ory Kratos Production

### DISCUSSION THREAD

**Discussion by Aditya51293** [ory/kratos#3150] [Q&A]
I am creating ory kratos with postgre SQL using deploy to production.
But it is asking for cerbot after the Nginx installation. But I am trying to do with local ip.
Is there a way to install it without SSL and port 4433 is not working.
https://www.ory.sh/docs/kratos/guides/deploy-kratos-example

**Comment by vinckr** [comment]
Hello @Aditya51293 

Yes, you can run Ory Kratos without SSL. However, it's important to note that running any application without SSL, especially one that handles authentication and user management, can pose serious security risks.

To run Ory Kratos without SSL, you need to make sure that the ports 4455, 4433, 4434, and 4436 are free. You should always use 127.0.0.1 as the hostname and never use localhost because browsers treat these two as separate domains and will have issues with setting and using cookies.

If port 4433 is not working, it might be because it's being used by another process or blocked by a firewall. You can check if the port is open or closed on a Linux server using the instructions provided in this [link](https://serverfault.com/questions/309052/check-if-port-is-open-or-closed-on-a-linux-server).
For the database, Ory Kratos supports SQLite, PostgreSQL, MySQL, and CockroachDB as database backends. In your case, you're using PostgreSQL. You need to change the dsn configuration in kratos.yml to use the Postgres database you configured before.
Finally, you can test your setup using the serve command. If everything is set up correctly, you should see a message indicating that the service has been started.

Please note that this setup is suitable for a quickstart or development environment. For a production environment, it's highly recommended to secure your application with SSL.
